notebooks/fixtures/config-exp/fv3-rap-physics.yaml

```
@@ -0,0 +1,6 @@
+task_make_ics:
+  chgres_cube:
+    namelist:
+      update_values:
+        config:
+          varmap_file: "{{ user.PARMdir }}/ufs_utils/varmap_tables/GSDphys_var_map.txt"
```

Before clicking commit here, I offer an alternative below.
```suggestion
          varmap_file: "/path/to/ufs-srweather-app/parm/ufs_utils/varmap_tables/GSDphys_var_map.txt"
```